modules/sdk/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties

```diff
@@ -1,17 +1,113 @@
-#liferay.workspace.bundle.token.download=false
-#liferay.workspace.bundle.token.email.address=
-#liferay.workspace.bundle.token.force=false
-#liferay.workspace.bundle.token.password=
-#liferay.workspace.bundle.token.password.file=
-#liferay.workspace.bundle.url=https://cdn.lfrs.sl/releases.liferay.com/portal/7.0.3-ga4/liferay-ce-portal-tomcat-7.0-ga4-20170613175008905.zip
-#liferay.workspace.default.repository.enabled=false
-#liferay.workspace.environment=local
-#liferay.workspace.home.dir=bundles
-#liferay.workspace.modules.default.repository.enabled=false
-#liferay.workspace.modules.dir=modules
-#liferay.workspace.plugins.sdk.dir=plugins-sdk
-#liferay.workspace.themes.dir=themes
-#liferay.workspace.wars.dir=wars
-
-microsoft.translator.client.id=
-microsoft.translator.client.secret=
\ No newline at end of file
+##
+## Liferay Workspace
+##
+
+    #
+    # Set this to true if the "liferay.workspace.bundle.url" property is set to
+    # a DXP bundle Zip. This property allows the token residing in the
+    # "~/.liferay" folder to be used to validate your user credentials when
+    # downloading the bundle. The default value is false.
+    #
+    #liferay.workspace.bundle.token.download=false
+
+    #
+    # Set the email address to use when downloading a DXP bundle. This must
+    # match the email address registered for your DXP subscription. Only use
+    # this property if you do not want to use a workspace token to supply your
+    # user credentials (i.e., liferay.workspace.bundle.token.download=false).
+    #
+    #liferay.workspace.bundle.token.email.address=
```

Comment:
The property `liferay.workspace.bundle.token.email.address` has no effect if `liferay.workspace.bundle.token.download=false`. The value must be the user's email address (for example, the one in www.liferay.com) that's used when creating a token, not when downloading a bundle.